src/lter/hbef/products.csv

```diff
@@ -8,3 +8,4 @@ prodcode,prodname,type,retrieve_status,munge_status,derive_status,notes
 100,rain_gauge_locations,spatial,ready,pending,NA,NA
 ms001,stream_flux,derived,NA,NA,ready,NA
 ms002,precip_flux,derived,NA,NA,pending,NA
+94,ws_boundary,spatial,ready,ready,NA,NA
```

Comment:
let's keep msXXX prodcodes at the end of the file, since they will sometimes depend on other derived products that must update first. I have it sorting alphanumerically by prodcode, which ensures msXXX codes run last for now, but that won't always be true. We could also make a custom sort function that works in a tidy pipeline.